packages/venia-concept/src/components/Mask/mask.js

```diff
@@ -11,14 +11,20 @@ class Mask extends Component {
             root_active: PropTypes.string
         }),
         dismiss: PropTypes.func,
-        isActive: PropTypes.bool
+        isActive: PropTypes.bool,
+        opacity: PropTypes.number
+    };
+
+    static defaultProps = {
+        opacity: 0.5
     };
 
     render() {
-        const { classes, dismiss, isActive } = this.props;
+        const { classes, dismiss, isActive, opacity } = this.props;
         const className = isActive ? classes.root_active : classes.root;
+        const style = isActive ? { opacity } : {};
```

Comment:
This has to be in CSS. Opacity is purely a presentational choice and has no business here.